### PR TITLE
[website][adhoc] Fix spellcheck errors and typos

### PR DESCRIPTION
**Pull request by aromanenko-dev**
Fix spellcheck errors and typos found with `cspell` tool

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by aromanenko-dev** [comment]
R: @kennknowles 
R: @pabloem

File: website/www/site/content/en/documentation/io/built-in/hcatalog.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ pipeline
 `HCatalogIO` is built for Apache HCatalog versions 2 and up and will not work out of the box for older versions of HCatalog.
 The following illustrates a workaround to work with Hive 1.1.
 
-Include the following Hive 1.2 jars in the über jar you build.
+Include the following Hive 1.2 jars in the uber jar you build.
```
Comment (kennknowles) [inline website/www/site/content/en/documentation/io/built-in/hcatalog.md:73]: This one was correct before, no? I don't know :-)

**Review by kennknowles — APPROVED** [review]
Wow, I'm surprised! especially the ones that were formal identifiers or the ones that impacted rendering or functionality.

File: website/www/site/content/en/documentation/io/built-in/hcatalog.md (hunk 1 above)
Comment (aromanenko-dev) [reply website/www/site/content/en/documentation/io/built-in/hcatalog.md:73]: Well, yes, [it comes from a German word](https://stackoverflow.com/a/11947093/10687325) with "ü" as the first character, whenever, iinm, in English we don't have such one. So actually both are correct but I thought that for English version we should stick with just English alphabet.

**Comment by aromanenko-dev** [comment]
@kennknowles 
> the ones that impacted rendering or functionality.

Actually, that was an initial reason that I decided to check the whole documentation for errors/typos.

**Pull request merged**